bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -5495,6 +5495,75 @@ int getContentChildrenCount(Content content) throws TskCoreException {
 		}
 	}
 
+	/**
+	 * Count of the content object's children of specified types. The types are
+	 * instances of TskData.TSK_FS_NAME_TYPE_ENUM types and the matching is
+	 * performed against the tsk_files.dir_type column. Some usage examples are
+	 * to get a count of all subdirectories, which requires searching for all
+	 * children of types TskData.TSK_FS_NAME_TYPE_ENUM.DIR OR
+	 * TskData.TSK_FS_NAME_TYPE_ENUM.VIRT_DIR.
+	 *
+	 * Note: this is generally more efficient then preloading all children and
+	 * counting, and facilities lazy loading.
```

Comment:
I realize you cut and pasted this from above, but I think it should be "facilitates"

Reply:
Fixed